cdb_test.go

```diff
@@ -44,6 +56,23 @@ func TestGet(t *testing.T) {
 	}
 }
 
+func TestEach(t *testing.T) {
+	db, err := cdb.Open("./test/test.cdb")
+	require.NoError(t, err)
+	require.NotNil(t, db)
+
+	n := 0
+	db.Each(func(key, value []byte) error {
+		assert.Equal(t, string(expectedKeyOrder[n]), string(key))
```

Comment:
this should just be `expectedRecords[n][0]` instead of using `expectedKeyOrder`.